packages/google-pay-integration/e2e/GooglePayBraintreeMockingResponses.ts

```diff
@@ -0,0 +1,72 @@
+
+export const orderPayment =
+    '{"status":"ok","three_ds_result":{"acs_url":null,"payer_auth_request":null,"merchant_data":null,"callback_url":null},"errors":[]}';
+
+export const payment =
```

Comment:
Is the `clientToken` long string necessary?

Reply:
This is necessary for correct initialization of the braintree client without errors. If we do not pass the correct `clientToken`, we will encounter the corresponding error and see the error modal window at initialization. It interferes with the success of the test.